tap_purecloud/__init__.py

```diff
@@ -333,6 +333,12 @@ def sync_user_schedules(api_instance: WorkforceManagementApi, config, unit_id, u
 
     if first_page:
         singer.write_schema('user_schedule', schemas.user_schedule, ['start_date', 'user_id'])
+        singer.write_schema('user_schedule_shift', schemas.user_schedule_shift, ['user_id', 'id', 'week_schedule__id'])
```

Comment:
Pkey change since the shift["id"] and user_id is not sufficient. Going to drop the table and full sync instead of altering the constraints.